src/amo/containers/Home.js

```diff
@@ -1,11 +1,45 @@
-import React from 'react';
+import React, { PropTypes } from 'react';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+
+import 'amo/css/Home.scss';
+
+export class HomePageBase extends React.Component {
+  static propTypes = {
+    i18n: PropTypes.object.isRequired,
+  }
 
-export default class Home extends React.Component {
   render() {
+    const { i18n } = this.props;
     return (
-      <div>
-        <h1>AMO Home Page Hello World</h1>
+      <div className="HomePage">
+        <h2 className="HomePage-h2">{i18n.gettext('What do you want Firefox to do?')}</h2>
+        <ul className="HomePage-cat-list">
+          <li className="HomePage-block-ads"><a href="#block-ads"><span>{i18n.gettext('Block Ads')}</span></a></li>
```

Comment:
Is the `<span>` needed? This should probably use `Link` and a stateless component could DRY it up a bit:

```jsx
const CatLink = ({ children, name, ...props }) => (
  <li className={`HomePage-${name}`}><Link {...props}>{children}</Link></li>
);
// ...
<ul className="HomePage-cat-list">
  <CatLink name="block-ads" href="#block-ads">{i18n.gettext('Block Ads')}</CatLink>
</ul>